apps/desktop/desktop/src/main/java/bisq/desktop/main/content/bisq_easy/open_trades/trade_state/TradeDataHeader.java

```diff
@@ -245,32 +404,47 @@ private Triple<Text, Text, VBox> getElements(@Nullable String description) {
             descriptionLabel.getStyleClass().add("bisq-easy-open-trades-header-description");
             Text valueLabel = new Text();
             valueLabel.getStyleClass().add("bisq-easy-open-trades-header-value");
-            VBox.setMargin(descriptionLabel, new Insets(2, 0, 1.5, 0));
+            VBox.setMargin(descriptionLabel, new Insets(0, 0, 5, 0));
             VBox vBox = new VBox(descriptionLabel, valueLabel);
-            vBox.setAlignment(Pos.CENTER_LEFT);
+            vBox.setAlignment(Pos.TOP_LEFT);
             vBox.setMinHeight(HEIGHT);
             vBox.setMaxHeight(HEIGHT);
             return new Triple<>(descriptionLabel, valueLabel, vBox);
         }
 
-        private Triple<Triple<Text, Text, Text>, HBox, VBox> getAmountElements() {
+        private Triple<Triple<Text, Node, Text>, HBox, VBox> getAmountElements() {
             Text descriptionLabel = new Text();
             descriptionLabel.getStyleClass().add("bisq-easy-open-trades-header-description");
-            Text amount = new Text();
-            amount.getStyleClass().add("bisq-easy-open-trades-header-value");
+            BitcoinAmountDisplay btcAmountText = new BitcoinAmountDisplay("0");
```

Comment:
Use `BitcoinAmountDisplay bitcoinAmountDisplay`. When renaming a class IntelliJ asks you if you want to rename fields with the matching name.